diff --git a/modules/luci-mod-system/htdocs/luci-static/resources/view/system/system.js b/modules/luci-mod-system/htdocs/luci-static/resources/view/system/system.js
--- a/modules/luci-mod-system/htdocs/luci-static/resources/view/system/system.js
+++ b/modules/luci-mod-system/htdocs/luci-static/resources/view/system/system.js
@@ -214,11 +214,12 @@ return view.extend({
 		o.uciconfig = 'luci';
 		o.ucisection = 'main';
 		o.ucioption = 'lang';
-		o.value('auto');
+		o.value('auto', _('Automatic'));
+		o.value('en', 'English');
 
 		var k = Object.keys(uci.get('luci', 'languages') || {}).sort();
 		for (var i = 0; i < k.length; i++)
-			if (k[i].charAt(0) != '.')
+			if (k[i].charAt(0) != '.' && k[i] != 'en')
 				o.value(k[i], uci.get('luci', 'languages', k[i]));
 
 		o = s.taboption('language', form.ListValue, '_mediaurlbase', _('Design'))
